metadata-ingestion/src/datahub/ingestion/source/gc/dataprocess_cleanup.py

```diff
@@ -277,7 +277,17 @@ def delete_dpi_from_datajobs(self, job: DataJobEntity) -> None:
         assert self.ctx.graph
 
         dpis = self.fetch_dpis(job.urn, self.config.batch_size)
-        dpis.sort(key=lambda x: x["created"]["time"], reverse=True)
+        try:
+            dpis.sort(key=lambda x: x["created"]["time"], reverse=True)
```

Comment:
What about having something like this:
```
        dpis.sort(
            key=lambda x: x["created"]["time"]
            if x["created"] and x["created"]["time"]
            else 0,
            reverse=True,
        )
```

In this case, you don't need all the try-catch, and these entities will surely be deleted because it has created time 0.